### PR TITLE
PMM-12223 added test for the clickhouse version

### PR DESCRIPTION
**Pull request by vyarosh**
please note that test will fail until https://github.com/percona/pmm/pull/2489 will be merged

### CODE REVIEW

**Comment by vyarosh** [comment]
Test result are local only:

File: pmm-tests/check_upgrade.py [hunk 1]
```diff
@@ -60,6 +61,11 @@ def main():
         verify_command(f"docker exec {pmm_server_docker_container} rpm -qa | grep percona-qan-api2-{args.version}")
         verify_command(
             f"docker exec {pmm_server_docker_container} rpm -qa | grep percona-dashboards-{args.version}")
+
+        ### PMM-12223 - Verify Clickhouse  is v22.8 or later since 2.41.0
```
Comment (saikumar-vs) [inline pmm-tests/check_upgrade.py:65]: look like u mean v23.8 in the comment ?

**Review by saikumar-vs — APPROVED** [review]
Thanks for working on this. 
A minor comment. please feel free to fix and push once PR is merged.

**Review by BupycHuk — APPROVED** [review]

File: pmm-tests/check_upgrade.py (hunk 1 above)
Comment (vyarosh) [reply pmm-tests/check_upgrade.py:65]: Actually, no - it's a copy of the Ticket's title: see https://jira.percona.com/browse/PMM-12223 
But I suppose you're right and I will change the it be more accurate in the sibling PR: https://github.com/percona/pmm-qa/pull/645

**Pull request merged**